src/Typesense/TypesenseDefaultOptions.cs

```diff
@@ -0,0 +1,42 @@
+using System;
+using System.IO;
+
+namespace Squadron
+{
+    /// <summary>
+    /// Default RavenDB resource options
+    /// </summary>
+    public class TypesenseDefaultOptions : ContainerResourceOptions
+    {
+        /// <summary>
+        /// Configure resource options
+        /// </summary>
+        /// <param name="builder"></param>
+        public override void Configure(ContainerResourceBuilder builder)
+        {
+            var password = Guid.NewGuid().ToString("N");
```

Comment:
I would use a static API key. Makes debugging a lot easier. 
imagine debugging an issue where you need to connect to typesense. 
It's much simpler when the container has the same api key every time

Reply:
Good point. Changed to static default value